CHANGELOG.md

```diff
@@ -34,6 +34,22 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
     - `getMessageGranules()` - Get the granules from a Cumulus execution message, if any.
   - Added `@cumulus/common/cloudwatch-event/isFailedSfStatus()` to determine if a Step Function status from a Cloudwatch event is a failed status
 
+- **CUMULUS-639**
+  - Adds SAML JWT and launchpad token authentication to Cumulus (configurable)
+    - **NOTE** to authenticate with Launchpad ensure your launchpad user_id is in the `<prefix>-UserTable`
+    - when Cumulus configured to protect API via Launchpad:
+         - New endpoints
+            - `GET /samlLogin` - starting point for SAML SSO creates the login request url and redirects to the SAML Identidy Provider Service (IDP)
```

Comment:
Is it better to use `GET /saml/login`?  just to be consistent with `/saml/auth`

Reply:
I originally just did /samlLogin because it was a slight difference from the dashboard POV for the login button. I was thinking of this in terms of /saml/this /saml/that. I don't think it matters

Reply:
I don't have strong opinions. But I like the suggestion if it's not too hard to update the nams requests.

Reply:
Oh, no nams required, I'll fix.